week1/armstrong.rb

```diff
@@ -9,5 +9,36 @@
 # and `false` otherwise.
 
 def armstrong_number(n)
-    raise NotImplementedError # Delete this line and start coding here
+
+    sum = 0
+    origional_no = n
+
+    len = n.to_s.length
+
+    if n < 0 
+        puts("false")
+        
+    else    
+        
+        while n != 0
+        digit = n%10
+        sum += digit.pow(len)
+        n = n/10
+        end
+
+        if sum == origional_no
+        puts("true")
+        else
+        puts("false")
+        end
+
+    end 
+
 end
+
+puts("Enter number")
+n = gets.chomp.to_i
+armstrong_number(n)
+
+end
```

Comment:
Hey @praffulgupta, good job with resolving conflicts, assuming this is your first time!

Some corrections-
1. Since this is an auto-graded assignment, you'd only need to modify code within the function block. Lines 39-43 are not necessary and can be removed as they interfere with the auto-grading. To test if your function works correctly, you can [run the tests locally by following these instructions](https://github.com/IRIS-NITK/IRIS-RoR-Bootcamp-2021/tree/main/week1#run-test-locally)
2. We check for [return](http://ruby-for-beginners.rubymonstas.org/writing_methods/return_values.html) values when testing with certain test cases to grade your submission. Hence, instead of the print statements (`puts("...")`), you can use [`return`](https://medium.com/rubycademy/the-return-keyword-in-ruby-df0a7f578fcb) statements.

Reply:
yes sir, it's my first time ..i have done all required changes. can u please review it once again

Reply:
yes no problem